docs/api.rst

```diff
@@ -0,0 +1,48 @@
+Python API
+==========
+
+Tracing
+-------
+
+Classes
+^^^^^^^
+.. autoclass:: opentracing.Tracer
+   :members:
+
+.. autoclass:: opentracing.ReferenceType
+   :members:
+
+.. autoclass:: opentracing.Reference
+   :members:
+
+Utility Functions
+^^^^^^^^^^^^^^^^^
+.. autofunction:: opentracing.start_child_span
+
+.. autofunction:: opentracing.child_of
+
+.. autofunction:: opentracing.follows_from
+
+Spans
```

Comment:
The ReST structure here is:
```
Python API
 |- Tracing
 |   |- Classes
 |   |   |- opentracing.Tracer
 |   |   |- opentracing.ReferenceType
 |   |   `- opentracing.Reference
 |   `- Utility Functions
 |       |- opentracing.start_child_span
 |       |- opentracing.child_of
 |       `- opentracing.follows_from
 |- Spans
 |   |- opentracing.Span
 |   `- opentracing.SpanContext
 |- Propagation
 |   `- opentracing.Format
 `- Exceptions
     |- opentracing.InvalidCarrierException
     |- opentracing.SpanContextCorruptedException
     `- opentracing.UnsupportedFormatException
```

I should probably remove the *Classes* and *Utility Functions* headings or revisit the structure altogether.  I chose the top-level heading names based on the modules themselves.